<launch>
    <arg name="xbot2_config" default="$(find centauro_config)/centauro_basic.yaml"/>

    <include file="$(find centauro_gazebo)/launch/centauro_world.launch" pass_all_args="true"/>

    <node pkg="kyon_controller" type="xbot2_core" name="xbot2" output="screen"
        args="--hw sim --simtime --config $(arg xbot2_config)"/>
</launch>
